pkg/splunk/enterprise/afwscheduler.go

```diff
@@ -159,6 +167,9 @@ func (ppln *AppInstallPipeline) transitionWorkerPhase(worker *PipelineWorker, cu
 		replicaCount = 1
 	}
 
+	// Disable the  existing worker, so that either it can be safely transitioned to new pipeline or can act as a base for fan-out workers
+	makeWorkerInActive(worker)
```

Comment:
It was there before at the end, as part of the call `setContextForNewPhase `. Now, that has fewer steps, as we can use it across multiple places. So, moved into a separate function, and also called at the beginning, so that, it can work as a seed worker for fanOut logic.